js/index.js

```diff
@@ -1,23 +1,35 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity input');
+  const priceValue = price.innerHTML;
+  const quantityValue = quantity.value;
```

Comment:
Here you can simplify code, to make it even shorter. By inserting line 4 code directly instead of word 'price' on line 6. So below syntax is equivalent to what you have:

```
  const priceValue = product.querySelector('.price span').innerHTML;
  const quantityValue = product.querySelector('.quantity input').value;
```